site/docs/components/svg/fundSvg.tsx

```diff
@@ -0,0 +1,16 @@
+export const fundSvg = (
+    <svg
+      width="30"
+      height="30"
```

Comment:
We usually set these to:

```suggestion
      width="100%"
      height="100%"
```

And use the parent component to narrow down sizing!